src/__tests__/utils.js

```diff
@@ -38,6 +38,20 @@ export const getBookmark = (id) => {
 	}]
 }
 
+export const getConditionalBookmark = (id) => {
```

Comment:
this is a generic lib, we don't want to put in our exact use cases, make this more generic, even in the test utils, so function name something like getNamedBookmark, and change COND to something like 'PREFIX::'

Reply:
we can replace the above getBookmark function with this new one. then just use:
```
export const getBookmark = (id, prefix = 'PREFIX::') => {
```

Reply:
don't want to duplicate and have both.